# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/wayland/moz.build
# Commit: 6770fb6b8d92
# Full Hash: 6770fb6b8d9217b7d75ed7169cb1389c868498ed
# Author: Martin Stransky <stransky@redhat.com>
# Date: 2024-09-06 21:37:32
# Regressor Bug: 1915596
# File Overlap Count: 2
# Description:
#   Bug 1915596 [Wayland] Get touchpad hold event directly from Wayland display r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D220551
# ==============================================================================

diff -r 791b8e439bbd -r 6770fb6b8d92 widget/gtk/wayland/moz.build
--- a/widget/gtk/wayland/moz.build	Fri Sep 06 08:37:00 2024 +0000
+++ b/widget/gtk/wayland/moz.build	Fri Sep 06 09:16:28 2024 +0000
@@ -12,6 +12,7 @@
     "idle-inhibit-unstable-v1-protocol.c",
     "linux-dmabuf-unstable-v1-protocol.c",
     "pointer-constraints-unstable-v1-protocol.c",
+    "pointer-gestures-unstable-v1-protocol.c",
     "relative-pointer-unstable-v1-protocol.c",
     "viewporter-protocol.c",
     "xdg-activation-v1-protocol.c",
@@ -24,6 +25,7 @@
     "idle-inhibit-unstable-v1-client-protocol.h",
     "linux-dmabuf-unstable-v1-client-protocol.h",
     "pointer-constraints-unstable-v1-client-protocol.h",
+    "pointer-gestures-unstable-v1-client-protocol.h",
     "relative-pointer-unstable-v1-client-protocol.h",
     "viewporter-client-protocol.h",
     "xdg-activation-v1-client-protocol.h",